dg/media/app_dashboards/js/loop_dashboard.js

```diff
@@ -2809,7 +2818,8 @@ function aggregator_payment_sheet(data_json, aggregator) {
 
     var aggregator_payment = payments_data.aggregator_data;
     var transport_payment = payments_data.transportation_data;
-    var gaddidar_payment = payments_data.gaddidar_data;
+    var gaddidar_contribution_data = payments_data.gaddidar_data;
+    // var gaddidar_payment = payments_data.gaddidar_data;
 
     var sno = 1;
     var str1 = "Rs. ";
```

Comment:
Why Rs. instead of symbol when symbol is everywhere else ? Also, Can global variable be used ?